src/main/java/deadline.java

```diff
@@ -0,0 +1,24 @@
+public class deadline extends task {
+
+    protected String date;
+
+    public deadline(String name, Integer number, boolean isDone, String taskType) {
```

Comment:
Avoid using primitive types in Java.
```suggestion
    public deadline(String name, Integer number, Boolean isDone, String taskType) {
```